datafiles/templates/UserSignupReset/SignupConfirmation.email.st

```diff
@@ -7,10 +7,11 @@ this link:
   $confirmlink$
 
 After your account is created, you will need to email the Hackage
-Trustees at hackage-trustees@haskell.org requesting to be added to the
+Trustees at hackage-trustees@haskell.org requesting addition to the
 uploader group. In this email, please include your Hackage username
-and a package that you'd like to upload. (This measure is
-unfortunately necessary to prevent spam accounts.)
+and a link to the package you'd like to upload (on a host like gitlab
+or github). This measure is unfortunately necessary to prevent spam
```

Comment:
Do we want to add an "if possible" here?

Reply:
I don't think so, it's table stakes for an opensource project and if it's under a hackage-compatible licence anyone could stand one up.